README.md

```diff
@@ -1 +1,41 @@
-# Bubble_Sort
\ No newline at end of file
+# Bubble Sort
+
+> First Ruby Project at Microverse to be introduced to working with Ruby.
+
+![microverse](https://camo.githubusercontent.com/3a5835d4f56c57cec85939ac345e43fef164c178/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f4d6963726f76657273652d626c756576696f6c6574)
+
+## Description
+
+This project is about creating 2 different sorting functions 'sort' and 'sort_by' which may take an array as parameter, and sort it in orders based on the Project Requirements.
+
+
+## Built With
+
+- Ruby
+- Linter
+- VScode
+
+## Contributing
+
+Contributions, issues and feature requests are welcome! Start by:
+
+  - Forking the project
+  - Cloning the project to your local machine
+  - cd into the project directory
+  - Run git checkout -b your-branch-name
+  - Make your contributions
+  - Push your branch up to your forked repository
+  - Open a Pull Request with a detailed description to the development branch of the original project for a review
+
+
+
+## Author
+
+👤 Suyash Fowdar
+Github: [@Krishnzzz](https://github.com/krishnzzz)
+
+👤 Azeez Akinkunmi
+Github: [@kunmi02](https://github.com/kunmi02)
```

Comment:
[Optional] You can add more social network links such as (linkedin and twitter) if you have them